src/libcrun/status.c

```diff
@@ -390,6 +398,13 @@ libcrun_read_container_status (libcrun_container_status_t *status, const char *s
       return crun_make_error (err, 0, "'created' missing in %s", file);
     status->created = xstrdup (YAJL_GET_STRING (tmp));
   }
+  {
+    const char *owner[] = { "owner", NULL };
+    tmp = yajl_tree_get (tree, owner, yajl_t_string);
+    if (UNLIKELY (tmp == NULL))
+      return crun_make_error (err, 0, "'owner' missing in %s", file);
```

Comment:
I think in this case we just need to ignore `owner` instead of raising an error.

```
status->owner = tmp ? xstrdup (YAJL_GET_STRING (tmp)) : NULL;
```